docusaurus/docs/dev-docs/installation/docker.md

```diff
@@ -12,6 +12,10 @@ import DockerEnvTable from '/docs/snippets/docker-env-table.md'
 Strapi does not build any official container images. The following instructions are provided as a courtesy to the community. If you have any questions please reach out on [Discord](https://discord.strapi.io).
 :::
 
+:::warning
```

Comment:
```suggestion
:::danger
```

This will turn this into a callout with the highest level of importance (a red one).

Reply:
Sounds great! will commit that